measures/ResStockArguments/measure.rb

```diff
@@ -844,14 +844,45 @@ def run(model, runner, user_arguments)
     args[:electric_panel_breaker_spaces_type] = 'headroom'
     args[:electric_panel_breaker_spaces] = breaker_spaces_headroom
 
-    # FIXME
+    # Assign miscellaneous permanently connected appliance loads
     if args[:electric_panel_load_other_power].nil?
       args[:electric_panel_load_other_power] = 0
     end
-    microwave_power = [0, 100, 200, 300].sample(1, random: Random.new(args[:building_id]))
-    garbage_disposal_power = [0, 200, 400, 600].sample(1, random: Random.new(args[:building_id]))
-    args[:electric_panel_load_other_power] += microwave_power[0]
-    args[:electric_panel_load_other_power] += garbage_disposal_power[0]
+    # Assume all homes have a microwave
+    if args[:geometry_unit_num_occupants] <= 2
+      microwave_power = 900 # W, small, <= 0.9 cu ft
+    elsif args[:geometry_unit_num_occupants] <= 4
```

Comment:
@joseph-robertson - is this okay or do I need to define the condition as `args[:geometry_unit_num_occupants] > 2 && args[:geometry_unit_num_occupants] <= 4`?

Reply:
I think so. Microwave exists in unoccupied homes too.
I am also wondering if we should do this based on # of bedrooms instead. Is there a conversion we can use between occupants and bedrooms?